api/src/pcapi/connectors/entreprise/backends/api_entreprise.py

```diff
@@ -290,6 +290,13 @@ def get_siren(self, siren: str, with_address: bool = True, raise_if_non_public:
             closure_date=self._convert_timestamp_to_date(siren_data.get("date_cessation")),
         )
 
+    def _format_ape_code(self, ape_code: str | None) -> str | None:
+        if not ape_code:
+            return None
+        if ape_code[2] == ".":
+            return ape_code.replace(".", "")
+        return ape_code
```

Comment:
Pas besoin de tester avant `replace` la présence du motif, je pense que tu peux simplifier ainsi : 
```suggestion
        return ape_code.replace(".", "")
```